src/id/oploverz/src/eu/kanade/tachiyomi/animeextension/id/oploverz/Oploverz.kt

```diff
@@ -147,6 +149,39 @@ class Oploverz : ConfigurableAnimeSource, ParsedAnimeHttpSource() {
 
     override fun videoUrlParse(document: Document) = throw Exception("not used")
 
+    private fun Elements.ordered(): Elements {
```

Comment:
isnt this completely useless?
you are adding at the end of the list so this doesnt change the list at all
just remove it, no?

Reply:
Uhhh suree

Reply:
Im coping the otakudesu list sorter though

Reply:
But sure ig